src/vos/vos_iterator.c

```diff
@@ -480,6 +483,9 @@ set_reprobe(vos_iter_type_t type, unsigned int acts,
 	case VOS_ITER_OBJ:
 		if (yield || (delete && (type == VOS_ITER_OBJ)))
 			anchors->ia_reprobe_obj = 1;
+	case VOS_ITER_COUUID:
```

Comment:
(style)  Possible switch case/default not preceeded by break or fallthrough comment